### PR TITLE
Getting the repo ready for the 3.0.27 release.

### PR DESCRIPTION
**Pull request by jfboismenu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +53,16 @@ Integration and unit tests are provided.
 
 ## Changelog
 
-**v3.0.27 - TBD**
+**v3.0.27 - 2016 Feb 17**
+
+   + Make sure HTTP proxy authentication works with the @ character in a password.
+   + Make sure sudo authentication test works with Shotgun versions after 6.3.10
+   + Smarter uploading of thumbnails and filmstrips with the upload() method.
+   + Improve Travis build integration of the Python-API to run the full suite of
```
Comment (kporangehat) [inline README.md]: I don't think this is really necessary as a release note since it's just for us internally right?

**Comment by kporangehat** [comment]
One comment, then ready to go! :shipit:

File: README.md (hunk 1 above)
Comment (jfboismenu) [inline README.md]: Well, it's still nice to advertise that we are more thorough. :)

**Comment by neilvictorgrey** [comment]
Hey @jfboismenu I'm closing off this PR since 3.0.27 was released without it being merged. Feel free to reopen if there's something still active here that I'm missing.

**Pull request closed without merging**